esf_main.tex

```diff
@@ -300,11 +307,19 @@ \subsubsection{Description/additional circuitry}
 
 \subsubsection{Wiring}
 %Describe the wiring, show schematics including the circuit board, show data regarding the cables and connectors used.  If not detailed in section 2.1, be sure to show how the device open the shutdown circuit.
-TODO
+
+TODO: Which board is this connected to with the relay on it? 
```

Comment:
This board will have the relay on it. Shutdown already runs to the bulkhead, so it's not an issue if this board interrupts it.